### PR TITLE
updates to group comparison saving

### PR DESCRIPTION
**Pull request by adamabeshouse**
(1) Immediately save new session when adding/deleting a group
(2) Save selected groups and drag order only to URL

### CODE REVIEW

File: src/pages/groupComparison/GroupComparisonStore.ts [hunk 1]
```diff
@@ -70,56 +70,100 @@ export default class GroupComparisonStore {
...
+    @action public updateGroupOrder(oldIndex:number, newIndex:number) {
+        let groupOrder = this.groupOrder;
+        if (!groupOrder) {
+            groupOrder = this._originalGroups.result!.map(g=>g.name);
+        }
+        groupOrder = groupOrder.slice();
+        const poppedUid = groupOrder.splice(oldIndex, 1)[0];
+        groupOrder.splice(newIndex, 0, poppedUid);
+
+        (window as any).routingStore.updateRoute({ groupOrder: JSON.stringify(groupOrder) } as Partial<GroupComparisonURLQuery>);
```
Comment (kalletlak) [inline src/pages/groupComparison/GroupComparisonStore.ts]: may be join by `,`

Comment (adamabeshouse) [reply src/pages/groupComparison/GroupComparisonStore.ts]: a group name could have a comma in it

Comment (alisman) [reply src/pages/groupComparison/GroupComparisonStore.ts]: lets pass in routingStore from view instead of using window global.

Comment (adamabeshouse) [reply src/pages/groupComparison/GroupComparisonStore.ts]: done

**Pull request merged**